app/screens/thread/thread.test.js

```diff
@@ -8,7 +8,7 @@ import Preferences from 'mattermost-redux/constants/preferences';
 import {General, RequestStatus} from 'mattermost-redux/constants';
 import PostList from 'app/components/post_list';
 
-import Thread from './thread.js';
+import Thread from './thread.ios';
```

Comment:
Since this test seems to be specific now to iOS, I'd suggest to explicitly name it so we know that we needed to add a separate test for Android.  Could be something like,
- test file as `thread.ios.test.js`
- component name as `ThreadIos`

Reply:
Is it really? I could’ve done the same but for Android instead, I just picked a filename to replace the old one but if you think that the snapshot test is that important I’ll make the changes and add Android as well. 0/5 here

Reply:
Oh right, if no other test specific to Android other than snapshot, then not worth adding. My comment was only about the name change suggestion, 0/5 as well.